app/config.py.example

```diff
@@ -45,6 +45,15 @@ email_conf = ConnectionConfig(
     USE_CREDENTIALS=True,
 )
 
+# import
+NUM_OF_VALUES = 4  # Event contains head, content, start_date and end_date.
+MAX_FILE_SIZE_MB = 5  # 5MB
+VALID_FILE_EXTENSION = (".txt", ".csv", ".ics")  # Can import only these files.
+VALID_YEARS = 20  # Events must be within 20 years range from the current year.
+EVENT_HEADER_NOT_EMPTY = 1  # 1- for not empty, 0- for empty.
+EVENT_HEADER_LIMIT = 50  # Max characters for event header.
+EVENT_CONTENT_LIMIT = 500  # Max characters for event characters.
+MAX_EVENTS_START_DATE = 10  # Max Events with the same start date.
```

Comment:
Not sure if these constants should be in the config file, but even if they should, names such as NUM_OF_VALUES or VALID_YEARS are pretty much ambiguous.

Reply:
Thank you Gonzom